src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -238,12 +268,12 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
   }
 
   "handleRequest" should "return an error if the dynamo entry does not have a type of 'asset'" in {
-    stubGetRequest(dynamoGetResponse.replace("Asset", "Folder"))
+    stubGetRequest(dynamoGetResponse.replace("Asset", "ArchiveFolder"))
     stubQueryRequest(emptyDynamoQueryResponse)
     val ex = intercept[Exception] {
       TestLambda().handleRequest(standardInput, outputStream, null)
     }
-    ex.getMessage should equal(s"Object $assetId is of type Folder and not 'asset'")
+    ex.getMessage should equal(s"Object $assetId is of type ArchiveFolder and not 'asset'")
```

Comment:
Should "asset" here and in the test name be capitalised?

Reply:
Ok, I've updated the exception in the lambda code as well.